addons/project/views/project_views.xml

```diff
@@ -749,19 +749,19 @@
             <field name="model">project.project</field>
             <field name="arch" type="xml">
                 <form string="Project">
-                    <div class="oe_title">
+                    <div class="oe_title mw-100 pe-0">
                         <label for="name" string="Name"/>
                         <h1>
-                            <field name="name" class="o_project_name oe_inline" placeholder="e.g. Office Party"/>
+                            <field name="name" class="o_project_name" placeholder="e.g. Office Party"/>
                         </h1>
                     </div>
                     <field name="user_id" invisible="1"/>
-                    <div class="row mt16 o_settings_container"/>
-                    <div name="alias_def" colspan="2" attrs="{'invisible': [('alias_domain', '=', False)]}">
-                        <label for="alias_name" class="oe_inline mt-4" string="Create tasks by sending an email to"/>
+                    <div class="row mt-2 o_settings_container" invisible="1"/>
+                    <div name="alias_def" class="mt-4" colspan="2" attrs="{'invisible': [('alias_domain', '=', False)]}">
+                        <label for="alias_name" string="Create tasks by sending an email to"/>
                         <field name="alias_enabled" invisible="1"/>
                         <span>
-                            <field name="alias_name" class="oe_inline" placeholder="e.g. office-party"/>@<field name="alias_domain" class="oe_inline" readonly="1" />
+                            <field name="alias_name" placeholder="e.g. office-party"/>@<field name="alias_domain"/>
```

Comment:
Why isn't it readonly anymore? It might be needed, but it doesn't look like an alignment issue.

Reply:
Because `alias_name` is compute/read-only field so it always will be read-only so I think we don't explicitly need to add `readonly` attribute in xml. Thanks :)

Reply:
makes sense! thank you :)